### ISSUE TITLE
Classification Results on OmniObject3D from ModelNet40-trained Models

### ISSUE THREAD

**Issue by jingema99** [omniobject3d/OmniObject3D#37]
Thank you for your great work.
I have two quick questions about the evaluation results in Table 2 of the OmniObject3D paper:

1. I noticed that ModelNet40 and OmniObject3D share a number of classes, possibly around a dozen. Does the data presented in Table 2 exclusively pertain to the performance on these shared classes?
2. When evaluating the ModelNet40-trained models on OmniObject3D, was the testing data limited to the test split of OmniObject3D?

**Comment by wutong16** [comment]
Hi,
1. The results in Table 2 is only based on the shared categories between OmniObject3D and ModelNet40. Please find the category mapping [here](https://github.com/omniobject3d/OmniObject3D/issues/7#issuecomment-1763337003).
2. When evaluating the pretrained models on ModelNet40, all the data in OmniObject3D under the shared categories are used as test set since there is no training set for this specific setting.

**Comment by tangsankou** [comment]
> Hi,
> 
> 1. The results in Table 2 is only based on the shared categories between OmniObject3D and ModelNet40. Please find the category mapping [here](https://github.com/omniobject3d/OmniObject3D/issues/7#issuecomment-1763337003).
> 2. When evaluating the pretrained models on ModelNet40, all the data in OmniObject3D under the shared categories are used as test set since there is no training set for this specific setting.

I downloaded the dataset from the link https://opendatalab.com/OpenXDLab/OmniObject3D-New/tree/main/raw/point_clouds/hdf5_files/1024, but I didn't find any data for the 'desk' category, and there are only 24 objects for the 'table' category. Could you please check these? Thank you for your assistance.
